src/staking-v3/components/dapp-selector/ModalSelectDapp.vue

```diff
@@ -189,10 +189,21 @@ export default defineComponent({
   border: 1px solid $navy-1;
   padding: 16px;
   margin-bottom: 24px;
-
   input {
     outline: none;
     width: 100%;
   }
 }
+
+.body--dark {
+  .search {
+    border-color: $gray-1;
+    input {
+      background-color: $navy-1;
+    }
+  }
+  //   .selection--order {
```

Comment:
Let's remove commented code

Reply:
@impelcrypto I've deleted it in [8947a54](https://github.com/AstarNetwork/astar-apps/pull/1078/commits/8947a540b381f8e47297d76349ba06c603d443a6)